admin-panel/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "properties": [
+        {
+            "name": "cryptic.admin-panel.api-token",
+            "type": "java.lang.String",
+            "description": "Cryptic server api-token."
+        },
+        {
+            "name": "cryptic.admin-panel.server-url",
+            "type": "java.lang.String",
+            "description": "Cryptic server url."
+        }
+    ]
+}
```

Comment:
I think this is generated at compile time. -> No need to push it.

Reply:
https://github.com/cryptic-game/java-backend/blob/master/admin-panel/build.gradle#L24 This Annotation processor does it.